### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 18)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

File: azureadps-2.0/AzureAD/Remove-AzureADMSApplicationKey.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Removes a key credential from an application'
   code: |-
-    PS C:\>Remove-AzureADMSApplicationKey -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -KeyId "FDA27CF-1B58-4CAE-8CE7-CD04F0AAB945" -Proof {token}
+    PS C:\>Remove-AzureADMSApplicationKey -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -KeyId "FDA27CF-1B58-4CAE-8CE7-CD04F0AAB945" -Proof {token}
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Remove-AzureADMSApplicationKey.yml:14]:
```suggestion
    PS C:\>Remove-AzureADMSApplicationKey -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -KeyId "aaaaaaaa-0b0b-1c1c-2d2d-333333333333" -Proof {token}
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md [hunk 2]
```diff
@@ -31,26 +31,26 @@ ObjectId                             AppId                                Displa
 421c3f21-22b1-43ea-b438-f00bcad54bd7 f9009add-63a4-4231-9532-9bdc22742922 PowerShellGraphAPI
 4862738f-9ce9-4db6-ab55-e185049f4597 d961ff63-d659-42d5-8ca8-908b3bbb79cb WingTips
 49a8bc01-2751-450b-a2e8-b4267f609513 10d861e6-90b3-4854-a504-f656aab2a14e AzurePopulator
-79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md:34]: rogue object id. 
```suggestion
79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md [hunk 3]
```diff
@@ -31,26 +31,26 @@ ObjectId                             AppId                                Displa
...
 9c4fb233-e88c-4a61-acc9-e8fdcb6758dd e5e29b8a-85d9-41ea-b8d1-2162bd004528 Tenant Schema Extension App
 a5fd58ca-9f1b-4184-ba7c-2595b5831e21 641e422d-29af-49c9-a24e-c0ee05ff10d5 PowerShellRunner
 c4fdf87f-f68e-4859-8bcf-36579b66005e 71715b24-8cdd-432b-a138-86e8ad179274 Woodgrove HR App
 d58d399f-56c3-409c-9efc-fdc28a6bd50e 3ad57eaf-2547-4161-81ae-fde64b5e1c0f ExtensionAttributes
 e9cfe5ad-c9eb-4cd7-87c2-2a69059aeb69 576ea3a9-3d7f-4bcc-a2b5-2d1a5088075e GraphDirectoryExtension
 
 
-PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
+PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md:42]: Related to above, fixing in rerun
```suggestion
PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md [hunk 4]
```diff
@@ -31,26 +31,26 @@ ObjectId                             AppId                                Displa
...
 PS C:\WINDOWS\system32> Get-AzureADDeletedApplication
 
 ObjectId                             AppId                                DisplayName
 --------                             -----                                -----------
-79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md:47]:
```suggestion
79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md [hunk 5]
```diff
@@ -31,26 +31,26 @@ ObjectId                             AppId                                Displa
...
 
-PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
+PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md:49]:
```suggestion
PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md [hunk 6]
```diff
@@ -31,26 +31,26 @@ ObjectId                             AppId                                Displa
...
 
 ObjectId                             AppId                                DisplayName
 --------                             -----                                -----------
-79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.md:53]:
```suggestion
79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml [hunk 7]
```diff
@@ -24,26 +24,26 @@ examples:
...
-    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
 
-    PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
+    PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
 
     ObjectId                             AppId                                DisplayName
     --------                             -----                                -----------
-    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml:46]:
```suggestion
    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml [hunk 8]
```diff
@@ -24,26 +24,26 @@ examples:
...
+    PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
     PS C:\WINDOWS\system32> Get-AzureADDeletedApplication
 
     ObjectId                             AppId                                DisplayName
     --------                             -----                                -----------
-    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
 
-    PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
+    PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml:42]:
```suggestion
    PS C:\WINDOWS\system32> Restore-AzureADDeletedApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml [hunk 9]
```diff
@@ -24,26 +24,26 @@ examples:
...
 
 
-    PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
+    PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
     PS C:\WINDOWS\system32> Get-AzureADDeletedApplication
 
     ObjectId                             AppId                                DisplayName
     --------                             -----                                -----------
-    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml:40]:
```suggestion
    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml [hunk 10]
```diff
@@ -24,26 +24,26 @@ examples:
...
     9c4fb233-e88c-4a61-acc9-e8fdcb6758dd e5e29b8a-85d9-41ea-b8d1-2162bd004528 Tenant Schema Extension App
     a5fd58ca-9f1b-4184-ba7c-2595b5831e21 641e422d-29af-49c9-a24e-c0ee05ff10d5 PowerShellRunner
     c4fdf87f-f68e-4859-8bcf-36579b66005e 71715b24-8cdd-432b-a138-86e8ad179274 Woodgrove HR App
     d58d399f-56c3-409c-9efc-fdc28a6bd50e 3ad57eaf-2547-4161-81ae-fde64b5e1c0f ExtensionAttributes
     e9cfe5ad-c9eb-4cd7-87c2-2a69059aeb69 576ea3a9-3d7f-4bcc-a2b5-2d1a5088075e GraphDirectoryExtension
 
 
-    PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
+    PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml:35]:
```suggestion
    PS C:\WINDOWS\system32> Remove-AzureADApplication -ObjectId 79592454-dea7-4660-9d91-f1768e5055ac
```

File: azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml [hunk 11]
```diff
@@ -24,26 +24,26 @@ examples:
     421c3f21-22b1-43ea-b438-f00bcad54bd7 f9009add-63a4-4231-9532-9bdc22742922 PowerShellGraphAPI
     4862738f-9ce9-4db6-ab55-e185049f4597 d961ff63-d659-42d5-8ca8-908b3bbb79cb WingTips
     49a8bc01-2751-450b-a2e8-b4267f609513 10d861e6-90b3-4854-a504-f656aab2a14e AzurePopulator
-    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Restore-AzureADDeletedApplication.yml:27]:
```suggestion
    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

File: azureadps-2.0/AzureAD/Remove-AzureADMSConditionalAccessPolicy.md [hunk 12]
```diff
@@ -31,7 +31,7 @@ Conditional access policies are custom rules that define an access scenario.
 
 ### Example 1: Deletes a conditional access policy in Azure AD by PolicyId.
 ```
-PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
+PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Remove-AzureADMSConditionalAccessPolicy.md:34]:
```suggestion
PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

File: azureadps-2.0/AzureAD/Remove-AzureADMSConditionalAccessPolicy.yml [hunk 13]
```diff
@@ -8,7 +8,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Deletes a conditional access policy in Azure AD by PolicyId.'
   code: |-
-    PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
+    PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Remove-AzureADMSConditionalAccessPolicy.yml:11]:
```suggestion
    PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

File: azureadps-2.0/AzureAD/Remove-AzureADMSNamedLocationPolicy.md [hunk 14]
```diff
@@ -24,7 +24,7 @@ Named locations are custom rules that define network locations which can then be
 
 ### Example 1: Deletes a named location policy in Azure AD with given PolicyId.
 ```
-PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
+PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Remove-AzureADMSNamedLocationPolicy.md:27]:
```suggestion
PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
```

File: azureadps-2.0/AzureAD/Remove-AzureADMSNamedLocationPolicy.yml [hunk 15]
```diff
@@ -8,7 +8,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Deletes a named location policy in Azure AD with given PolicyId.'
   code: |-
-    PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
+    PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Remove-AzureADMSNamedLocationPolicy.yml:11]:
```suggestion
    PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
```

File: azureadps-2.0/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.md [hunk 16]
```diff
@@ -24,7 +24,7 @@ Delete an Azure Active Directory permission grant condition set object by id.
 
 ### Example 1: Delete a permission grant condition set from a policy
 ```
-PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
+PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.md:27]:
```suggestion
PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
```

File: azureadps-2.0/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.yml [hunk 17]
```diff
@@ -18,7 +18,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Delete a permission grant condition set from a policy'
   code: |-
-    PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
+    PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.yml:21]:
```suggestion
    PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru, I've double checked and seen that policy id is no longer a sensitive term. I remembered there being an issue in another repo about that.

**Review by msewaweru — APPROVED** [review]

**Pull request merged**